website/src/get-started/wordcount-example.md

```diff
@@ -1362,7 +1378,11 @@ PCollection<KV<String, Long>> wordCounts = windowedWords.apply(new WordCount.Cou
 ```
 
 ```py
-# This feature is not yet available in the Beam SDK for Python.
+import apache_beam as beam
+from apache_beam.transforms import window
+...
+...
+word_counts = windowed_words | beam.Map(WordCount()) 
```

Comment:
I think this example expects a composite transform - but I don't think wer'e using composites in wordcount.py: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/wordcount.py#L96-L102

Perhaps something like so:

```suggestion
word_counts = windowed_words | wordcount.CountWords()
```

although in this case, `CountWords` does not really exist. I am not sure how to represent this example...

Reply:
@pabloem you are right.  We can probably provide the implementation in this example for the clarification. Another way of doing is, providing the actual implementation of word count and provide the link here as we do for Java example.
Let me know which one you like. I am fine either way.

Reply:
it could be something like this: 
```
class CountWordsFn(Dofn):
   def process(self, element):
        # Add logic here to split line into words.
        # call metrics to increment value.

  class CountWords(PTransform):

    def expand(self, pcoll):
      return pcoll |  FlatMap(CountWordsFN())
```

Reply:
Using the same thing for the actual wordcount example makes sense to me.